http/src/commonMain/kotlin/com/mirego/trikot/http/requestPublisher/ResponseHttpRequestPublisher.kt

```diff
@@ -0,0 +1,22 @@
+package com.mirego.trikot.http.requestPublisher
+
+import com.mirego.trikot.http.HttpConfiguration
+import com.mirego.trikot.http.HttpHeaderProvider
+import com.mirego.trikot.http.HttpRequestFactory
+import com.mirego.trikot.http.HttpResponse
+import com.mirego.trikot.http.HttpResponseException
+import com.mirego.trikot.http.RequestBuilder
+
+open class ResponseHttpRequestPublisher(
```

Comment:
I wonder if `DeserializableHttpRequestPublisher` should not inherit from `ResponseHttpRequestPublisher` so HttpCode processing would not be dupplicated. 

This would mean that 
```
processResponse() {
  response = successfulResponse(super.processResponse(...)) 
}
```

Reply:
That makes sense